pyproject.toml

```
@@ -1,11 +1,44 @@
 [build-system]
-requires = [
-    "setuptools",
-]
-
-[tool.pytest.ini_options]
-minversion = "6.0"
-testpaths = [
-    "whatrecord/tests",
-]
-asyncio_mode = "auto"
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
```

Maybe a higher dev status here
```suggestion
classifiers = [ "Development Status :: 5 - Production/Stable", "Natural Language :: English", "Programming Language :: Python :: 3",]
```